amplitude.js

```diff
@@ -5864,6 +5865,29 @@ AmplitudeClient.prototype._apiKeySet = function _apiKeySet(methodName) {
  */
 AmplitudeClient.prototype._loadSavedUnsentEvents = function _loadSavedUnsentEvents(unsentKey) {
   var savedUnsentEventsString = this._getFromStorage(localStorage$1, unsentKey);
+  var events = this._parseSavedUnsentEventsString(savedUnsentEventsString, unsentKey);
+
+  var savedUnsentEventsStringLegacy = this._getFromStorageLegacy(localStorage$1, unsentKey);
+  var legacyEvents = this._parseSavedUnsentEventsString(savedUnsentEventsStringLegacy, unsentKey);
+
+  var unsentEvents = events.concat(legacyEvents);
+
+  // Migrate legacy events out of storage
+  this._removeFromLegacyStorage(localStorage$1, unsentKey);
+  this._setInStorage(localStorage$1, unsentKey, JSON.stringify(unsentEvents));
+
+  return unsentEvents;
+};
+
+AmplitudeClient.prototype._removeFromLegacyStorage = function _removeFromLegacyStorage(storage, key) {
```

Comment:
Nonblocker: maybe don't put these legacy helper methods on the prototype? We don't need to give users access to them

Reply:
Going to punt on this one until a bigger refactor. The library is already using the crude underscore prefix means "private" convention.